fix: modify remove_filtered_policy method to support filtering of empty strings

sanguinedab opened the pull request:
Modify remove_filtered_policy method to support filtering of empty strings

such as
`enforcer.remove_filtered_policy(0, "alice", "", "read") # The obj can be an empty string representing any`

And I looked at the official documentation and other SDKs (e.g. golang version)
![image](https://github.com/user-attachments/assets/c79c1677-ea9b-4aac-b6ac-f1d1b3c4f85d)

sanguinedab committed:
Modify remove_filtered_policy method to support filtering of empty strings

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/pycasbin/pymongo-adapter?pullRequest=12) <br/>All committers have signed the CLA.

leeqvip commented:
@sanguinedab Please fix
![image](https://github.com/user-attachments/assets/954723a0-7615-4b66-b5ca-1d93b8ae7570)

sanguinedab committed:
Update adapter.py

fix format

sanguinedab commented:
> Please fix ![图像](https://private-user-images.githubusercontent.com/35752209/379117870-954723a0-7615-4b66-b5ca-1d93b8ae7570.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk2NjM3MDUsIm5iZiI6MTcyOTY2MzQwNSwicGF0aCI6Ii8zNTc1MjIwOS8zNzkxMTc4NzAtOTU0NzIzYTAtNzYxNS00YjY2LWI1Y2EtMWQ5M2I4YWU3NTcwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDIzVDA2MDMyNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ2M2M5NzZkZDRkNGVhYzY3ZTc1ZGJkZDgzZWEwZjA4OGFhM2E2NGZlYzk2NGRlY2MxZmQ0NGY0ZmY4MjAyNDAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.jbbXqp0k72Ns-c7hfLb_KL3ZSPFEiQ-_P7Xd7dB2MFQ)
Ok, I fixed this format

leeqvip commented:
@sanguinedab The linter still fails

sanguinedab committed:
fix adapter.py black format

sanguinedab commented:
> @sanguinedab The linter still fails

Sorry I fixed it

Does the Semantic PR specification need to be changed, sorry I don't know much about this specification, I changed my title with reference to other successful PRs
